### ISSUE TITLE
Update ArcGIS online item description and Metadata XML file everytime the metadata cache is rebuilt

### ISSUE THREAD

**Issue by thomas-maschler** [wri/gfw-sync#57]

**Comment by thomas-maschler** [comment]
you can use this code to update AGOL metadata items

https://gist.github.com/thomas-maschler/388554ddb6deae6371db

to update descriptions, title, tags, or credits simply add the new values to the data payload

```
d = {"overwrite": "true",
         "token" :token,
         "f":"json",
         "description": "new description goes here",
         "title": " New title goes here",
         "tags": "Comma, separated, list, of , tags",
         "licenseInfo": "License goes here",
         "accessInformation": "Copyright/ source goes here"
 }
```

**Comment by astrong19** [comment]
Hey Thomas, I noticed that this code works for updating metadata that is on root in the "my content" folder of ArcGIS Online. But it can't find metadata items that are stored in folders in the "my content" section of ArcGIS Online. Any thoughts on why?

**Comment by thomas-maschler** [comment]
That's weird. I would have thought it only uses the item ID. Maybe a bug?

**Comment by thomas-maschler** [comment]
Folders seem to be tricky to access.
https://geonet.esri.com/thread/114696

Can you check the item ID in the console to see if they use a different one when you update items manually?

**Comment by astrong19** [comment]
Yeah it's definitely weird, and might be a bug. The format of the Request urls look the same for items in a folder vs items on root. 

In the conservation folder: http://gfw.maps.arcgis.com/sharing/rest/content/items/04d892c083f54c638228931da081467b/info/metadata/metadata.xml

On root: http://gfw.maps.arcgis.com/sharing/rest/content/items/8262022e776d499b8c0eadd33406d9eb/info/metadata/metadata.xml
